route53-infrastructure.tf

```diff
@@ -53,6 +53,20 @@ resource "aws_route53_record" "service_loadbalancer_record_alb" {
   }
 }
 
+resource "aws_route53_record" "service_loadbalancer_record_alb_global_accelerator" {
+  count = local.infrastructure_ecs_cluster_services_alb_enable_global_accelerator && local.enable_infrastructure_route53_hosted_zone ? 1 : 0
+
+  zone_id = aws_route53_zone.infrastructure[0].zone_id
+  name    = "ga.${local.infrastructure_route53_domain}."
+  type    = "A"
```

Comment:
Since we have enabled dual stack higher up do we want to add a AAAA record as well. or does this do the right thing.

Reply:
Ah yup, I'll add a AAAA
I should add a AAAA for the service CloudFront too ...

Reply:
Since cloudfront CNAMEs i think AWS just does that.